src/Main.cc

```diff
@@ -155,6 +155,17 @@ void OnDisconnect(uWS::WebSocket<false, true>* ws, int code, std::string_view me
     ws->close();
 }
 
+void OnDrain(uWS::WebSocket<false, true>* ws) {
+    uint32_t session_id = static_cast<PerSocketData*>(ws->getUserData())->session_id;
+    Session* session = sessions[session_id];
+    if (session) {
+        spdlog::debug("Client {} [{}] WebSocket backpressure drains, remaining buffered amount: {} (bytes)", session->GetId(),
+            session->GetAddress(), ws->getBufferedAmount());
+    } else {
+        spdlog::debug("Unknown client WebSocket backpressure drains, remaining buffered amount: {} (bytes)", ws->getBufferedAmount());
```

Comment:
I would change these to "Draining WebSocket backpressure: client {} [{}]. Remaining buffered amount: {} (bytes)."